lib/warnings.pm

```diff
@@ -5,7 +5,7 @@
 
 package warnings;
 
-our $VERSION = "1.51";
+our $VERSION = "1.53";
```

Comment:
I think only one bump is needed here?

Reply:
blead is already at 1.52, so I went to 1.53.   Should it be 1.52?

Reply:
Well it didn't before but 1.52 is now in the released dev version 5.35.2 so 1.53 is needed. This will also need to be rebased on blead.

Reply:
And you should probably also squash the commits when rebasing

Reply:
I've squashed my commits.   It should ff-merge cleanly onto the warnings-default branch.   I would suggest we do that, close this PR, and and any further refinements to https://github.com/Perl/perl5/pull/18666

Reply:
I rebased my branch

Reply:
Rebased mine as well.